### ISSUE TITLE
Missing/Broken Setting

### ISSUE THREAD

**Issue by ghost** [A1eNaz/Xenia-Game-Settings#1]
Hey, I tried the settings that you provided (use_dedicated_xma_thread = false and use_new_decoder = true) for SEGA Superstar Tennis, but everytime I try to play, sound cuts out at random time, sometimes during the intro, sometimes during a match. Do you know a possibile fix? I tried with other games, but only here appears this issue. Do you need to know my PC config?

**Comment by A1eNaz** [comment]
try turning off `use_new_decoder` as some games like DiRT 2 and TDU2 break with it enabled.

**Comment by ghost** [comment]
I tried, but now the game remains stucked during the intro. Feel free to
reply when you have time, if you want I can provide you the xenia log

Il giorno mar 31 dic 2024 alle ore 16:09 AleNaz ***@***.***>
ha scritto:

**Comment by A1eNaz** [comment]
you can maybe try `license_mask = 1`  or `scribble_heap = true` which could boot up the game without the new decoder

**Comment by ghost** [comment]
Nope. I tried these days also using other settings found on YouTube, I
think it's for my PC, probably there's a component not much compatible,
btw, thank you for all

Il giorno mar 31 dic 2024 alle ore 19:13 AleNaz ***@***.***>
ha scritto:
